CHANGELOG.md

```diff
@@ -5,6 +5,23 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [5.0.1] - 2019-02-25
+
+### Features
+- Implement a new `typed-enum` rule to detect enum value that do not respect specified type [#913](https://github.com/stoplightio/spectral/pull/913)
+
+### Changed
```

Comment:
This could be added then,. Changed is for breaking changes, i.e. something you were using works differently now. Making a convenient new way to do something, but still supporting the old approach is an Added.